docs/cloud/features/observability/overview.md

```diff
@@ -1,22 +1,67 @@
+######### Is anything before Marisa's draft section (titled "how it works" below) worth keeping?
+
 # Overview
```

Comment:
this seciton feels like it should be on the Tobiko Cloud overview page, not on the observability page